packages/utils/src/logger.ts

```diff
@@ -72,3 +73,7 @@ export const attachConsoleErrorToPino = () => {
     }),
   )
 }
+
+if (process.env.PRINT_LOG) {
```

Comment:
If `LOG_LEVEL` or `DEBUG` is specified this should probably be enabled. I'd prefer if we use this as a false override.

e.g. by default not log output. If `DEBUG` then print output at log level `debug`. If `DEBUG` and `PRINT_LOG=false` then don't print output.

That said I don't see a case for this, it might make more sense to replace
```
if (process.env.PRINT_LOG)
```
with
```
if (process.env.DEBUG || process.env.LOG_LEVEL)
```

Reply:
https://github.com/zkopru-network/zkopru/pull/257/commits/c012d5aaf995b3954e214bd957e92bea50ff39e2 resolves this comment

Reply:
https://github.com/zkopru-network/zkopru/pull/257/commits/dc52ce2429977b4b829b79ea480d2c95d74442e7 resolves this comment